src/CompilerOptions.ts

```diff
@@ -41,3 +43,42 @@ export enum LuaTarget {
     Lua53 = "5.3",
     LuaJIT = "JIT",
 }
+
+export function validateOptions(options: CompilerOptions): ts.Diagnostic[] {
+    const diagnostics: ts.Diagnostic[] = [];
+
+    if (options.luaBundle && (options.luaBundleEntry === "" || !options.luaBundleEntry)) {
+        diagnostics.push(configErrorDiagnostic(`'luaBundleEntry' is required when 'luaBundle' is enabled.`));
+    }
+
+    if (options.luaBundle && options.luaLibImport === LuaLibImportKind.Inline) {
+        diagnostics.push(
+            configWarningDiagnostic(
+                `Using 'luaBundle' with 'luaLibImport: "inline"' might generate duplicate code. ` +
+                    `It is recommended to use 'luaLibImport: "require"'`
+            )
+        );
+    }
```

Comment:
Can you think of any cases where someone actually would want to keep using inline? I'm thinking we might just error in such case

Reply:
We use it on the playground